server/mrt.go

```diff
@@ -46,6 +46,15 @@ func (m *mrtWriter) Stop() {
 	close(m.dead)
 }
 
+func (m *mrtWriter) isAddPathReceiveEnabled(address string, family bgp.RouteFamily) bool {
+	if p, ok := m.s.neighborMap[address]; !ok {
+		return false
```

Comment:
the loop goroutine can't access to m.s.neighborMap; a race with the main goroutine?